src/operations.jl

```diff
@@ -199,10 +207,12 @@ function Calibrate(o::OperationRequest)
     Calibrate(sys, o.timespan, priors, data, num_chains, num_iterations, calibrate_method, ode_method)
 end
 
+# Jadon Clugston: maybe we want separate callbacks for the optimizer and the ode solve?
```

Comment:
This may have been a key part I was missing in the original request for intermediate results.  I think we only care about the optimizer